models/nxos/lldp/nxos_lldp_global.yml

```diff
@@ -0,0 +1,62 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: | 
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: nxos
+RESOURCE: lldp
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: nxos_lldp_global
+  version_added: 2.9
+  short_description: Configure and manage Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+  description: This module configures and manages the Link Layer Discovery Protocol(LLDP) attributes on NX-OS platforms.
+  author: Adharsh Srivats Rangarajan (@adharshsrivatsr)
+  options:
+    config:
+      description:
+        - A list of link layer discovery configurations
+      type: list
+      elements: dict
+      suboptions:
+        holdtime:
+          description:
+            - Amount of time the receiving device should hold the information (in seconds)
+          type: int
+        port-id: 
+          description:
+            - This attribute defines if the interface names should be advertised in the long(0) or short(1) form.
+          type: int
+          choice: [0,1]
+          default: 0
+        reinit: 
+          description:
+            - Amount of time to delay the intialization of LLDP on any interface (in seconds)
+          type: int
+        timer:
+          description:
+            - Frequency at which LLDP updates need to be transmitted (in seconds)
+          type: int
+        tlv_remove:
```

Comment:
Make this `tlv_select` and have the choices as suboptions for tlv_select and replace `-` by `_`.
```
        tlv_select:
          description:
            - 
          type: dict
          suboptions:
            dcbxp:
              description:
              type: bool
            port_vlan:
              description:
              type: bool
            management_address:
              description:
              type: dict
              suboptions:
                v4:
                  description:
                  type: bool
.
.
.
.
```